src/azul/chalice.py

```diff
@@ -40,6 +40,22 @@ def __init__(self, app_name, unit_test=False, spec=None):
             self._specs: Optional[MutableJSON] = None
         super().__init__(app_name, debug=config.debug > 0, configure_logs=False)
 
+    def _bearer_token(self, request: Request) -> Optional[str]:
+        try:
+            auth_header = request.headers.get('Authorization') or request.headers['authorization']
```

Comment:
HTTP header names are case-insensitive so HTTP client and server implementations typically use dedicated, case-insensitive mappings to represent them. That seems to be the case for Chalice.